sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalysisContext.java

```diff
@@ -38,6 +38,7 @@
     private final Map<SubqueryExpression, Object> arrayExpressionsChildren = new IdentityHashMap<>();
 
     private boolean hasAggregates;
+    private boolean forceReturnMultipleRows;
```

Comment:
I'd give this a less "apocalyptic" name - maybe something along the lines of "isReturnMultiplRows".